python-package/xgboost/compat.py

```diff
@@ -47,6 +47,32 @@ class DataFrame(object):
 
     PANDAS_INSTALLED = False
 
+# cudf
+try:
+    from cudf.dataframe import DataFrame as GDF
```

Comment:
Could you use names based on cuDF, e.g., `CUDF` and `CUDF_COL`?

Reply:
Resolved